src/style.css

```diff
@@ -37,12 +26,17 @@
   box-sizing: border-box;
   box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
   border-radius: 4px;
+  font-family: Avenir-book, sans-serif;
```

Comment:
add `font-family` to html tag and here use `inherit`
https://developer.mozilla.org/en-US/docs/Web/CSS/inherit